workshop/content/docs/publishing/first.md

```diff
@@ -13,18 +13,18 @@ You can find this dataset in `workshop/exercises/data/free-wifi-florence.csv`.
 
 This exercise consists of two key steps:
 
-* adapt the `pygeoapi.config.yml` to define this dataset as an OGC API - Features *collection*
-* ensure that pygeoapi can find and connec to the data file
+* adapt the `workshop/exercises/pygeoapi.config.yml` to define this dataset as an OGC API - Features *collection*
```

Comment:
suggest:

* adapt `workshop/exercises/pygeoapi.config.yml` to define this dataset as an OGC API - Features *Collection*